src/js/components/Anchor/stories/Weight.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+
+import { Anchor, Box, Grommet } from 'grommet';
+import { grommet } from 'grommet/themes';
+
+const Weight = () => {
+  return (
+    <Grommet theme={grommet}>
+      <Box align="center" pad="large">
+        <Anchor href="#" label="Anchor no weight" />
```

Comment:
Saying 'no weight' is a little tricky since the theme is still applying `weight`, so how about changing it to `default weight`?
```suggestion
        <Anchor href="#" label="Anchor default weight" />
```